packages/x-charts-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts-pro",
-  "version": "7.0.0-beta.1",
+  "version": "7.0.0-beta.2",
```

Comment:
Is this correct? I'm not clear on how prereleases should be bumped.

Reply:
Yes, this is correct. 👍